packages/sdk/chainDeploy/helpers/liquidators/registry.ts

```diff
@@ -17,25 +17,48 @@ export const configureLiquidatorsRegistry = async ({
     deployer
   )) as ILiquidatorsRegistry;
 
-  const strategies: string[] = [];
-  const inputTokens: string[] = [];
-  const outputTokens: string[] = [];
+  {
+    const strategies: string[] = [];
+    const inputTokens: string[] = [];
+    const outputTokens: string[] = [];
 
-  for (const redemptionStrategy of chainIdToConfig[chainId].redemptionStrategies) {
-    const { strategy, outputToken, inputToken } = redemptionStrategy;
-    const redemptionStrategyContract = await ethers.getContract(strategy, deployer);
+    for (const redemptionStrategy of chainIdToConfig[chainId].redemptionStrategies) {
+      const { strategy, outputToken, inputToken } = redemptionStrategy;
+      const redemptionStrategyContract = await ethers.getContract(strategy, deployer);
 
-    strategies.push(redemptionStrategyContract.address);
-    inputTokens.push(inputToken);
-    outputTokens.push(outputToken);
+      strategies.push(redemptionStrategyContract.address);
+      inputTokens.push(inputToken);
+      outputTokens.push(outputToken);
+    }
+    const matching = await liquidatorsRegistry.callStatic.pairsStrategiesMatch(strategies, inputTokens, outputTokens);
+    if (!matching) {
+      const tx = await liquidatorsRegistry._resetRedemptionStrategies(strategies, inputTokens, outputTokens);
+      console.log("waiting for tx ", tx.hash);
+      await tx.wait();
+      console.log("_resetRedemptionStrategies: ", tx.hash);
+    } else {
+      console.log("no redemption strategies to configure in the liquidators registry");
+    }
   }
-  const matching = await liquidatorsRegistry.callStatic.pairsStrategiesMatch(strategies, inputTokens, outputTokens);
-  if (!matching) {
-    const tx = await liquidatorsRegistry._resetRedemptionStrategies(strategies, inputTokens, outputTokens);
+
+  {
+    // TODO check if same instead of updating every time
```

Comment:
do we want to address this? I think such a check will bring the `LiquidatorsRegistryExtension` closer to the code size limit